### ISSUE TITLE
Using RefForwardingComponent results in TS2693

### ISSUE THREAD

**Issue by jonathanloske** [atanasster/webpack-react-docgen-typescript#1]
I have a Button component that is of type `RefForwardingComponent`. Compiling it results in `TS2693: 'RefForwardingComponent' only refers to a type, but is being used as a value here.`

My component has the following code:

```tsx
import React, { RefForwardingComponent, forwardRef } from "react";

type ButtonProps = {
	appearance?: "primary" | "secondary" | "ghost";
	bigSize?: boolean;
	iconStandalone?: boolean;
	textStyle?: boolean;
} & JSX.IntrinsicElements["button"];

const Button: RefForwardingComponent<HTMLButtonElement, ButtonProps> = forwardRef(({
	appearance,
	bigSize,
	className,
	iconStandalone,
	textStyle,
	...otherProps
}, ref) => {
	const getButtonClassName = ({
		appearance,
		bigSize,
		className,
		iconStandalone,
		textStyle
	}) => {

	// ... mangling the props to build the className

	return <button
		className={className}
		ref={ref}
		{...otherProps}
	/>;
});

Button.defaultProps = {
	appearance: "ghost",
	bigSize: false,
	textStyle: false,
};

export { Button };
```

**Comment by jonathanloske** [comment]
I'm using a custom webpack config with a configuration as indicated in the README. Using Storybook 5.2.6.

**Comment by atanasster** [comment]
Its a known issue with `react-docgen-typescript` I believe i got a workaround to fix it in `0.9.5`

**Comment by jonathanloske** [comment]
It works, thank you! 🙏
